tools/build.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--clone_dir", required=True)
+    parser.add_argument("--conda_path")
+    parser.add_argument("--config_mode", default="Shipping")
+    parser.add_argument("--unreal_engine_dir", required=True)
+    parser.add_argument("--num_parallel_jobs", type=int, default=1)
+    parser.add_argument("--target_platform", required=True, help="Values accepted are Win64, Linux, Mac")
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--tag", required=True)
+    args = parser.parse_args()
+
+    assert args.target_platform in ["Linux", "Mac"] and args.conda_path
```

Comment:
Don't want to maintain these help strings for all of our scripts, so let's remove. And they're hard to keep up-to-date, e.g., why does the assert not match the help string?

Reply:
I agree that help strings are hard to keep up-to date. But, for the argument `--target_platform`, we need to provide from a fix set of strings such as 'Win64', 'Linux', 'Mac' as this is being passed to and interpreted by Unreal's build cmd. These values need not change so it might be helpful to have the help string in this case.

Another way would be to assign these strings (Win64, Linux, Mac) to a local variable as part of the code, based on `--target_platform`. I think help string way is better than this.

I did modify the assert statement.

Reply:
You could make the same argument for many of our command-line arguments. Let's remove the help string.

Reply:
Hmm, okay.

I'm handling the target_platform based on sys.platform, so we don't need `--target_platform` anymore